InternTrack.Core.Api.Tests.Unit/Services/Foundations/Interns/InternServiceTests.Logic.Modify.cs

```diff
@@ -0,0 +1,66 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Force.DeepCloner;
+using InternTrack.Core.Api.Models.Interns;
+using Moq;
+using Xunit;
+
+namespace InternTrack.Core.Api.Tests.Unit.Services.Foundations.Interns
+{
+    public partial class InternServiceTests
+    {
+        [Fact]
+        public async Task ShouldModifyInternAsync()
+        {
+            // given
+            int randomNumber = GetRandomNumber();
+            int randomDays = randomNumber;
+            DateTimeOffset randomDate = GetRandomDateTime();
```

Comment:
I don't see it is used to create new intern one line below. It was expected that this will be used to create random intern and setup datetimebroker to return its value when called `.GetCurrentDateTime()`

[Click here](https://github.com/ElbekDeveloper/CashOverflow/blob/master/CashOverflow.Tests.Unit/Services/Foundations/Jobs/JobServiceTests.Logic.Modify.cs) to see how we did this in one of our OSS projects

Reply:
lines 25-28 (I create new intern using randomDate )
lines 32-34 (dateTimeBroker returns randomDate)